__tests__/app.test.js

```diff
@@ -39,3 +41,43 @@ describe.only('GET/api/topics', () => {
         })
     })
 })
+describe('GET/api/articles/:artcile_id', () => {
+    test('200 status code - returns user requested article as an array with object articles that contain all its properties', () => {
+        const articleId = 2
+        return request(app)
+        .get(`/api/articles/${articleId}`)
+        .expect(200)
+        .then(({body}) => {
+            const {article} = body;
+            expect(body.article).toBeInstanceOf(Array)
```

Comment:
I wouldn't expect this to be an array - as we are only sending back one object I would expect {article : {article_id...etc}}. Only send back an array when the query asks for multiple items. This should be a quick refactor of your test and model.